packages/portfolio-contract/src/portfolio.exo.ts

```diff
@@ -39,195 +45,187 @@ export const DECODE_CONTRACT_CALL_RESULT_ABI = [
       },
     ],
   },
-] as const;
-harden(DECODE_CONTRACT_CALL_RESULT_ABI);
-
-export const supportedEVMChains: CaipChainId[] = [
-  'eip155:43114', // Avalanche
-  'eip155:8453', // Base
-  'eip155:1', // Ethereum
-] as const;
-harden(supportedEVMChains);
+];
 
 const OrchestrationAccountShape = M.remotable('OrchestrationAccount');
 const VowStringShape = M.any(); // Vow(M.string())
 const KeeperI = M.interface('keeper', {
   getLCA: M.call().returns(OrchestrationAccountShape),
   getPositions: M.call().returns(M.arrayOf(M.string())),
-  initAave: M.call(ChainShape).returns(),
-  initCompound: M.call().returns(),
-  initUSDN: M.call(OrchestrationAccountShape).returns(),
   getUSDNICA: M.call().returns(OrchestrationAccountShape),
-  getAaveAddress: M.call().returns(VowStringShape),
-  getCompoundAddress: M.call().returns(VowStringShape),
+  initAave: M.call(M.string()).returns(),
+  initCompound: M.call(M.string()).returns(),
+  initUSDN: M.call(M.remotable('OrchestrationAccount')).returns(),
+  initGmp: M.call(M.remotable('OrchestrationAccount')).returns(),
+  getRemoteAddress: M.call().returns(VowStringShape),
```

Comment:
I was confused when I wrote `const VowStringShape = M.any();`
`VowShape` actually distinguishes vows from other stuff. But we can't express "a vow for a string" in a pattern, so I **suggest** that we don't give the impression that we can:

```suggestion
  getRemoteAddress: M.call().returns(VowShape),
```